frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -10,22 +11,27 @@ export interface MultiColumnFieldRowProps {
   children: React.ReactNode;
   onDelete: () => void;
   isReadOnly?: boolean;
+  disableDeleteRow?: boolean;
 }
 
-const minusCircleIcon = (onDelete: () => void, toolTip?: string) => {
+const minusCircleIcon = (onDelete: () => void, toolTip?: string, disableDeleteRow?: boolean) => {
   return (
-    <div className="odc-multi-column-field__col--button">
+    <div className={cx('odc-multi-column-field__col--button', { 'is-disabled': disableDeleteRow })}>
       <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
       <span className="sr-only">{toolTip || 'Delete'}</span>
     </div>
   );
 };
 
-const renderMinusCircleIcon = (onDelete: () => void, toolTip?: string) => {
+const renderMinusCircleIcon = (
+  onDelete: () => void,
+  toolTip?: string,
+  disableDeleteRow?: boolean,
+) => {
   return toolTip ? (
-    <Tooltip content={toolTip}>{minusCircleIcon(onDelete, toolTip)}</Tooltip>
+    <Tooltip content={toolTip}>{minusCircleIcon(onDelete, toolTip, disableDeleteRow)}</Tooltip>
   ) : (
-    minusCircleIcon(onDelete)
+    minusCircleIcon(onDelete, 'Delete', disableDeleteRow)
```

Comment:
If you make `toolTip` the last param instead you can simplify to this, rather than duplicating the string `Delete`:
```
    minusCircleIcon(onDelete, disableDeleteRow)
```